pcbdraw/populate.py

```diff
@@ -244,11 +263,13 @@ def generate_image(boardfilename: str, side: str, components: List[str],
     plot_args += ["--filter", ",".join(components)]
     plot_args += ["--highlight", ",".join(active)]
     plot_args += [boardfilename, outputfile]
+    tmp_std = sys.stdout        # make a copy of stdout
```

Comment:
I understand that Click changes stdout. But I have two questions:
- why it matters?
- the comment "make a copy of stdout" adds no information. I can see you make it. But why? This is what should be in the comment.

Reply:
- in case there are any temporary debug print statements (yes I know the debugger exists, but at times it's easier for quick debug sessions)
- can iterate in the comment if it's worth keeping